native/src/main/java/io/ballerina/stdlib/http/api/HttpDispatcher.java

```diff
@@ -414,6 +415,11 @@ static BObject getCaller(Resource resource, HttpCarbonMessage httpCarbonMessage,
 
     static BObject createRequestContext(HttpCarbonMessage httpCarbonMessage) {
         BObject requestContext = ValueCreatorUtils.createRequestContextObject();
+        String authHeader = httpCarbonMessage.getHeader(AUTHORIZATION_HEADER);
+        if (Objects.nonNull(authHeader) && authHeader.contains("Bearer ")) {
```

Comment:
Can we use a constant for `"Bearer "`?

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-http/pull/1470/commits/461c0635c2a83486b45f6fa21243bf23fbeb8227